# Message corresponding to vpImagePoint class
float64 i      # i (respectively v) position in the image
float64 j      # j (respectively u) position in the image

